tasm-test/src/LayoutBenchmarkTest.cc

```diff
@@ -0,0 +1,198 @@
+#include "TiledVideoManager.h"
+#include "SemanticIndex.h"
+#include "VideoManager.h"
+#include "Tasm.h"
+#include <gtest/gtest.h>
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include <functional>
+
+using namespace tasm;
+
+#define VIDEO_PATHS {"/home/maureen/red_videos/red_6sec_2k.mp4", "/home/maureen/red_videos/red_900sec_2k.mp4"}
+#define LABEL_PATHS {"/home/maureen/NFLX_dataset/detections/labels/birdsincage.db", "/home/maureen/visualroad_tiling/labels_yolo/traffic-2k-001_yolo.db"}
+#define VIDEO_LABELS {"bird", "car"}
+#define VIDEO_NAMES {"birdsincage", "traffic-2k-001_yolo"}
+#define NUM_FRAMES {180, 27000}
+#define IDS {197, 8476}
```

Comment:
It would be helpful to add a comment specifying how we picked these IDs if it's important so future us knows how to update them if we modify the tests.